src/main/java/team/gif/robot/UI.java

```diff
@@ -3,6 +3,7 @@
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
 
+import static com.ctre.phoenix.GadgeteerUartClient.GadgeteerProxyType.Pigeon;
```

Comment:
Is this the correct import or should it be:
import team.gif.robot.subsystems.drivers.Pigeon;

Reply:
import team.gif.robot.subsystems.drivers.Pigeon; This is right. Sorry.